fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/LuceneStoredFieldsTest.java

```diff
@@ -114,7 +114,7 @@ void testInsertDocuments(boolean useOptimizedStoredFieldsFormat) throws Exceptio
                     assertDocCountPerSegment(directory, List.of("_0"), List.of(3));
                 }
                 assertTrue(timer.getCounter(LuceneEvents.Waits.WAIT_LUCENE_GET_STORED_FIELDS).getCount() > 1);
-                assertTrue(timer.getCounter(LuceneEvents.Counts.LUCENE_WRITE_STORED_FIELDS).getCount() >= 3);
+                assertTrue(timer.getCounter(LuceneEvents.SizeEvents.LUCENE_WRITE_STORED_FIELDS).getCount() >= 3);
```

Comment:
1. Yeah, I did realize that. However, I am leaving this as is given the context of this PR.
2. I am not sure about that. `LUCENE_WRITE_STORED_FIELDS` was used as a count and reported size to `LUCENE_WRITE_SIZE`. However, the size is not tested here.

Reply:
> I am not sure about that. LUCENE_WRITE_STORED_FIELDS was used as a count and reported size to LUCENE_WRITE_SIZE. However, the size is not tested here.

My point was just that because you changed how these size events are recorded, we should probably have coverage of the new functionality within the test. If you feel like the other tests here are enough for that, then it's probably fine